hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ContainerImporter.java

```diff
@@ -79,6 +79,16 @@ public ContainerImporter(ConfigurationSource conf, ContainerSet containerSet,
         ScmConfigKeys.OZONE_SCM_CONTAINER_SIZE_DEFAULT, StorageUnit.BYTES);
   }
 
+  public boolean isAllowedContainerImport(long containerID) {
+    if (importContainerProgress.contains(containerID)) {
+      return false;
+    }
+    if (containerSet.getContainer(containerID) != null) {
+      return false;
+    }
+    return true;
```

Comment:
Nit: can be simplified.

```suggestion
    return !importContainerProgress.contains(containerID) &&
        containerSet.getContainer(containerID) == null;
```

Also, please add `<>` in `new HashSet()`, missed in last change (1cf54f34af3).

https://github.com/apache/ozone/blob/4e95b702a996ec65f9be6f4d50a4025e55991d89/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ContainerImporter.java#L63-L64